manager/controlapi/service_test.go

```diff
@@ -15,6 +15,20 @@ import (
 	"google.golang.org/grpc/codes"
 )
 
+func createCustomSpec(name, runtime string) *api.ServiceSpec {
+	spec := createSpec(name, runtime, 0)
+	spec.Task.Runtime = &api.TaskSpec_Custom{
+		Custom: &api.CustomSpec{
+			Name: runtime,
+			Payload: &gogotypes.Any{
+				TypeUrl: "com.docker.custom.runtime",
```

Comment:
Should perhaps be `com.docker.swarmkit.custom.runtime` to be properly namespaced? (maybe then the `custom` becomes redundant?) and/or maybe there should be a `spec` in there somewhere?